src/abstractdataframe/abstractdataframe.jl

```diff
@@ -2494,6 +2624,9 @@ are identical (checked with `===`). Otherwise, if two columns share some part of
 memory but are not identical (e.g. are different views of the same parent
 vector) then `reverse!` result might be incorrect.
 
+$METADATA_FIXED
+(metadata having other styles is dropped from parent data frame)
```

Comment:
Fixed. In fact it was mostly correct (as many functions you probably saw delete rows, which is not allowed for `SubDataFrame`). But indeed I missed it for few functions.

It applies to `SubDataFrame`, but technically also to `DataFrame`, as `parent` of `DataFrame` is identity.

Reply:
OK. Even if technically correct for `DataFrame`, it's confusing for users, and most of them are interested in `DataFrame` rather than `SubDataFrame`. So I would say something like "Metadata having other styles is dropped (from parent data frame when `df` is a `SubDataFrame`)."

Reply:
OK - fixed